SWLOR.Game.Server/Feature/AbilityDefinition/MartialArts/ElectricFistAbilityDefinition.cs

```diff
@@ -0,0 +1,124 @@
+﻿//using Random = SWLOR.Game.Server.Service.Random;
+using System.Collections.Generic;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Enumeration;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.AbilityService;
+using static SWLOR.Game.Server.Core.NWScript.NWScript;
+
+namespace SWLOR.Game.Server.Feature.AbilityDefinition
+{
+    public class ElectricFistAbilityDefinition : IAbilityListDefinition
+    {
+        public Dictionary<Feat, AbilityDetail> BuildAbilities()
+        {
+            var builder = new AbilityBuilder();
+            ElectricFist1(builder);
+            ElectricFist2(builder);
+            ElectricFist3(builder);
+
+            return builder.Build();
+        }
+
+        private static string Validation(uint activator, uint target, int level)
+        {
+            var weapon = GetItemInSlot(InventorySlot.RightHand);
```

Comment:
I had to change the knuckles to act like a should due to animation concerns so I think you need to check the left hand for this instead.